<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>loginwindow</class>
 <widget class="QMainWindow" name="loginwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>loginwindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="button_signin">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>370</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 85, 255);
background-color: rgb(170, 170, 127);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_exit">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>370</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 85, 255);
background-color: rgb(170, 170, 127);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="user_line">
    <property name="geometry">
     <rect>
      <x>332</x>
      <y>239</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Titillium Web</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 0, 0);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="user_lable">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>240</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>STLiti</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>用户：</string>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>50</y>
      <width>351</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文琥珀</family>
      <pointsize>72</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>DBMS</string>
    </property>
   </widget>
   <widget class="QLabel" name="user_photo">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>230</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_register">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>370</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 85, 255);
background-color: rgb(170, 170, 127);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>注册</string>
    </property>
   </widget>
   <widget class="QLabel" name="code_lable">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>290</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>STLiti</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>密码：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="code_line">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>290</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Titillium Web</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 0, 0);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="code_photo">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>280</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="rememberCheckBox">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>330</y>
      <width>98</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>记住密码</string>
    </property>
   </widget>
   <zorder>button_signin</zorder>
   <zorder>button_exit</zorder>
   <zorder>user_lable</zorder>
   <zorder>title</zorder>
   <zorder>user_photo</zorder>
   <zorder>button_register</zorder>
   <zorder>code_lable</zorder>
   <zorder>code_photo</zorder>
   <zorder>user_line</zorder>
   <zorder>code_line</zorder>
   <zorder>rememberCheckBox</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(134, 134, 134);</string>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>登录界面</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
